src/components/application_manager/CMakeLists.txt

```diff
@@ -49,9 +49,9 @@ include_directories (
   ${COMPONENTS_DIR}/rpc_base/include/
   ${COMPONENTS_DIR}/interfaces
   ${CMAKE_BINARY_DIR}/src/components/
-  ${COMPONENTS_DIR}/include/
+#  ${COMPONENTS_DIR}/include/
   ${COMPONENTS_DIR}/policy/src/policy/include/
-  ${COMPONENTS_DIR}/policy/src/policy/policy_table/table_struct/
+#  ${COMPONENTS_DIR}/policy/src/policy/policy_table/table_struct/
```

Comment:
@AMikhniev , i`ll change it.
But please write comments in commit, not in "Files changed"